iron-icon.html

```diff
@@ -85,6 +85,7 @@
       vertical-align: middle;
 
       fill: currentcolor;
+      stroke: currentcolor;
```

Comment:
i think it would be best if both of these were custom properties `--iron-icon-fill-color` and `--iron-icon-stroke-color`, for example, which would have the default values as `currentcolor`. That was somebody could customize them after the fact. What do you think?